core/nodejs14Action/Dockerfile

```diff
@@ -47,18 +47,20 @@ RUN apt-get update && apt-get install -y \
     unzip \
     && rm -rf /var/lib/apt/lists/*
 
-# Add sources and copy the package.json to root container,
+# Copy the package.json to root container,
 # so npm packages from user functions take precendence.
-#
 WORKDIR /nodejsAction
-ADD  . /nodejsAction/
 COPY package.json /
 
 # Customize runtime with additional packages.
 # Install package globally so user packages can override.
-#
-RUN cd / && npm install --no-package-lock --production \
-    && npm cache clean --force
+RUN cd / \
+  && npm install --no-package-lock --omit=dev \
```

Comment:
--production doesn't omit dev?

Reply:
Apparently --omit=dev is the successor to --production.

https://stackoverflow.com/a/9276112/20114275

I didn't know this until seeing it in our code base and googling it. This is a good catch though. I copy and pasted the part of the Node 18 Dockerfile that downloaded deps into Node 14 after I changed it. I didn't realize they were out of sync. If you're comfortable with the change here though, I think it's fine for us to use this.

Reply:
Yeah, I changed this when doing the Node 18 runtime because I've gotten deprecation warnings with `--production`.

I would've kept `--production` here though. But if it works the same 🤷